src/Entity/Attributes/ClientEntity.php

```diff
@@ -34,12 +31,13 @@ abstract class ClientEntity implements \Stringable
     protected mixed $entity_name;
 
     #[ORM\Column(name: 'created_at', type: 'datetime', nullable: false)]
+    #[Gedmo\Timestampable(on: 'create')]
     private $createdAt;
 
     #[ORM\Column(name: 'update_at', type: 'datetime', nullable: false)]
+    #[Gedmo\Timestampable(on: 'update')]
     private $updateAt;
```

Comment:
Can we not use a Timestampable trait here ?

Reply:
Haha I did it in first, but we have `$updateAt` and not `$updatedAt`.
We could update property/column name, but I preferred to focus on the migration

Reply:
I think we should do it right now ;)